src/Desktop.Analyzers/Core/Helpers/SecurityDiagnosticHelpers.cs

```diff
@@ -164,6 +165,30 @@ public static int HasXmlReaderParameter(IMethodSymbol method, CompilationSecurit
             return HasSpecifiedParameter(method, xmlTypes, IsXmlReaderType);
```

Comment:
`HasSpecifiedParameter` returns an `int`. Rename `HasSpecifiedParameter` to `GetSpecifiedParameterIndex`